client/go/vespa/deploy.go

```diff
@@ -370,7 +370,12 @@ func uploadApplicationPackage(url *url.URL, opts DeploymentOpts) (int64, error)
 	if err := opts.Target.PrepareApiRequest(request, sigKeyId); err != nil {
 		return 0, err
 	}
-	response, err := util.HttpDo(request, time.Minute*10, serviceDescription)
+
+	var response *http.Response
+	util.Spinner("Uploading application package ...", func() error {
```

Comment:
You must assign the returned error here. `err = util.Spinner ...`

Reply:
`util.Spinner` doesn't return any type:

https://github.com/vespa-engine/vespa/blob/master/client/go/util/spinner.go#L22

If we change that, we'll need to change all other places we use the `Spinner`. 
Do you think that would be a better approach?

Reply:
Ok. I think it is a better approach. Fixed. :)

Tested and everything is working like charm.